cterasdk/core/setup.py

```diff
@@ -0,0 +1,232 @@
+import logging
+import time
+import re
+
+from .base_command import BaseCommand
+from .enum import ServerMode, SetupWizardStage, SetupWizardStatus, SlaveAuthenticaionMethod
+from ..common import Object
+from ..convert import toxmlstr
+from ..exception import CTERAException, HostUnreachable, ExhaustedException
+
+
+class Setup(BaseCommand):
+    """
+    Global Admin Setup APIs
+    """
+
+    def __init__(self, portal):
+        super().__init__(portal)
+        self.stage = None
+
+    def _current_stage(self):
```

Comment:
I guess _update_stage or _get_current_stage are better names, no?

Reply:
Changed to _get_current_stage